modules/structs/pull.go

```diff
@@ -21,8 +21,13 @@ type PullRequest struct {
 	Assignees          []*User    `json:"assignees"`
 	RequestedReviewers []*User    `json:"requested_reviewers"`
 	State              StateType  `json:"state"`
+	Draft              bool       `json:"draft"`
 	IsLocked           bool       `json:"is_locked"`
 	Comments           int        `json:"comments"`
+	ReviewComments     int        `json:"review_comments"`
```

Comment:
If it isn't clearly defined in the GitHub docs, at least we can define it clearly.
So, what about a comment?

Reply:
I found a definition about ReviewComments on GitHub's REST-Api documentation. So I added a shortened version of that.